gamemaker.py

```diff
@@ -1,12 +1,31 @@
 class GameMaker:
-  def __init__(self, database):
-    self.database = database
-    self.loggedin = false
-  
-  def login(self, name, password):
-    if (self.database.gameMakerCred = {name, password}):
-      self.loggedin = true
-  //TODO Implement checkStatus
-  def checkStatus(self):
-  //TODO Implement logout
-  def logout(name):
+    def __init__(self, database):
+        self.database = database
+
+    def login(self, name, password):
+        return_value = False
+        current_user = self.database.get_current_user()
+        maker_cred = self.database.get_game_maker_cred()
+        if current_user is None:
+            if maker_cred["maker"] == "maker" and maker_cred["password"] == password:
+                self.database.set_current_user(self)
+                return_value = True
+        return return_value
+
+    def check_status(self):
+        string = ''
+        teams = self.database.get_teams()
+        if len(teams) > 0:
+            for team in teams:
+                string += team.user_name + ' is at ' + team.landmark + ' with ' + team.penalties + ' penalties\n'
```

Comment:
username not user_name
Also we don't have landmarks or penalties yet so this might break/crash